SECURITY.md

```diff
@@ -0,0 +1,58 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We take the security of Boston Open Dev's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue
+2. Report via one of these secure channels (in order of preference):
+   - Send a private message on [Zulip](https://osdc.zulipchat.com/#narrow/stream/406743-boston) to the maintainers
+   - Use GitHub's private security advisory reporting
```

Comment:
Is this easy for Phil to set up, and easy for reporters to use?

If someone is reporting a vulnerability, chances are they already have a Github account right? This should probably be the highest preference over Zulip and Mastodon, if not the only option.

Reply:
For convenience, could link to: https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability

Reply:
You're right—GitHub could be the primary option since most reporters will already have an account, and it’s easy to set up with its **Security Advisories** feature. Zulip and Mastodon were included as backups for flexibility, but I’m fine with prioritizing GitHub as the main (or even only) reporting channel.

Reply:
I'm fine with not listing Zulip as a way to report.